src/hyperloglog.c

```diff
@@ -227,6 +231,19 @@ static int simd_enabled = 1;
 #define HLL_USE_AVX2 0
 #endif
 
+#ifdef __aarch64__
+static int simd_enabled = 1;
+#define HLL_USE_NEON (simd_enabled)
+#else
+#define HLL_USE_NEON 0
+#endif
+
+#if HAVE_X86_SIMD || defined(__aarch64__)
+#define SIMD_SUPPORTED 1
```

Comment:
Let's put the declaration of `static int simd_enabled = 1;` here to cover all kinds of SIMD, instead of declaring the variable in two places (one for AVX2 and one for NEON).